source/views/ScanPane.svelte

```diff
@@ -90,41 +91,21 @@
   }
 </script>
 
+<Navbar bind:Tabs={NavTabs} bind:current={currentTab} />
 <div class="container">
-  <h1>Green Machine</h1>
-  <nav>
-    <ul>
-      <li>
-        <button
-          onclick={() => {
-            tab = Tabs.Plugins;
-          }}>Plugins</button
-        >
-      </li>
-      <li>
-        <button
-          onclick={() => {
-            tab = Tabs.DOMSelection;
-          }}>DOM Selection</button
-        >
-      </li>
-    </ul>
-  </nav>
-  {#if tab === Tabs.Plugins}
+  <!--Select plugins-->
+  {#if currentTab == "plugins"}
     {#each selectedPlugins as { name }, index}
       <PluginSelect bind:checked={selectedPlugins[index].checked} {name} />
     {/each}
-  {:else if tab === Tabs.DOMSelection}
-    <form>
-      <label><input name="selection" type="radio" /> Specify targets</label>
-      <label><input name="selection" type="radio" /> Scan entire site</label>
-    </form>
-    <h2>Targets</h2>
-    <button>Add</button>
-    <CSSTarget></CSSTarget>
-    <CSSTarget></CSSTarget>
+
+    <!--DOM Selection and entire website scan-->
+  {:else if currentTab == "dom-selection"}
```

Comment:
hvordan kan det være at vi har fjernet det der enum "safety" Tabs.DOMSelection ting ting for at bruge strings

Reply:
og du er meget glad for == i stedet for ===

Reply:
Enums er very gut fix. Min C# hjerne tænker i ==